src/univariate/continuous/gamma.jl

```diff
@@ -105,7 +105,8 @@ function rand(rng::AbstractRNG, d::Gamma)
         # TODO: shape(d) = 0.5 : use scaled chisq
         return rand(rng, GammaIPSampler(d))
     elseif shape(d) == 1.0
-        return rand(rng, Exponential(d.θ))
+        θ = 
+        return rand(rng, Exponential{partype(d)}(scale(d)))
     else
         return rand(rng, GammaGDSampler(d))
```

Comment:
Since `GammaMTSampler` seems to be faster in all (arguably not very extensive) benchmarks above, we might want to change this to
```suggestion
        return rand(rng, GammaMTSampler(d))
```